packages/fxa-auth-server/docs/api.md

```diff
@@ -3008,6 +3020,42 @@ Returns SMS status for the current user.
   is in the specified country.
   <!--end-query-param-get-smsstatus-country-->
 
+### Subscriptions
+
+#### GET /subscriptions/plans
+
+:lock: HAWK-authenticated with session token
+<!--begin-route-get-subscriptionplans-->
```

Comment:
Just FYI, these HTML comments are a hangover from when this doc used to be auto-generated. But we ditched that recently because the script kept braking, so we can stop adding them now.